hydra/garaga/starknet/cli/verify.py

```diff
@@ -146,3 +146,51 @@ def verify_onchain(
     rich.print(
         f"[bold green]Check it out on[/bold green] {voyager_link_tx(network, invoke_result.hash)}"
     )
+
+
+def calldata(
+    system: Annotated[
+        ProofSystem,
+        typer.Option(help="Proof system", autocompletion=complete_proof_system),
+    ],
+    vk: Annotated[
+        Path,
+        typer.Option(
+            help="Path to the verification key JSON file",
+            file_okay=True,
+            dir_okay=False,
+            exists=True,
+            autocompletion=lambda: [],
+        ),
+    ],
+    proof: Annotated[
+        Path,
+        typer.Option(
+            help="Path to the proof JSON file",
+            file_okay=True,
+            dir_okay=False,
+            exists=True,
+            autocompletion=lambda: [],
+        ),
+    ],
+    public_inputs: Annotated[
+        Path,
+        typer.Option(
+            help="Path to the public inputs JSON file",
+            file_okay=True,
+            dir_okay=False,
+            exists=True,
+            autocompletion=lambda: [],
+        ),
+    ] = None,
+):
+    """Generate Starknet verifier calldata given a proof and a verification key."""
+    vk_obj = Groth16VerifyingKey.from_json(vk)
+    proof_obj = Groth16Proof.from_json(proof, public_inputs)
+
+    calldata = groth16_calldata_from_vk_and_proof(
+        vk=vk_obj,
+        proof=proof_obj,
+    )
+
+    print(" ".join([str(x) for x in calldata]))
```

Comment:
Does starkli only support calldata with with spaces, not commas ? 
Ie : printing the array "[1, 2, 3]" would not work ? Only "1 2 3" ?

Reply:
`starkli` seems to only parse arrays when they are formatted with spaces.
Otherwise you get `Error: Failed to create Felt from string`